client/src/components/Checkout.jsx

```diff
@@ -14,15 +32,30 @@ class Checkout extends React.Component {
         response.json()
       ))
       .then((roomData) => {
-        this.setState({ rooms: (roomData) });
+        this.setState({
+          isLoaded: true,
+          rooms: roomData[0],
+        });
       });
   }
 
   render() {
+    if (!this.state.isLoaded) {
+      return <div>Loading...</div>;
+    }
     return (
-      <div>You are the best</div>
+      <Wrapper>
+        <div>{this.state.rooms[0].cost} per night</div>
+        <div>{this.state.rooms[0].avg_review}avg reviews</div>
+        <div>{this.state.rooms[0].reviews}reviews</div>
+        <div>Dates</div>
+        <div>{this.state.rooms[0].guests} guests</div>
```

Comment:
Don't forget to remove this hard-code later on. Also, ES Lint prefers object destructuring here.